app/src/main/java/com/example/waterme/ui/WaterMeApp.kt

```diff
@@ -56,10 +62,18 @@ import java.util.concurrent.TimeUnit
 
 @Composable
 fun WaterMeApp(waterViewModel: WaterViewModel = viewModel(factory = WaterViewModel.Factory)) {
+    val layoutDirection = LocalLayoutDirection.current
     WaterMeTheme {
         Surface(
-            modifier = Modifier.fillMaxSize(),
-            color = MaterialTheme.colorScheme.background
+            modifier = Modifier
+                .fillMaxSize()
+                .statusBarsPadding()
+                .padding(
+                    start = WindowInsets.safeDrawing.asPaddingValues()
+                        .calculateStartPadding(layoutDirection),
+                    end = WindowInsets.safeDrawing.asPaddingValues()
+                        .calculateEndPadding(layoutDirection)
+                ),
```

Comment:
Could we add horizontal padding here?

Reply:
No because depending on how you rotate (left vs right) to landscape the padding could be different.